tests/kernel/wave/wave_e2e_test.py

```diff
@@ -29,7 +29,7 @@ def test_copy(shape):
     wave_size = 64
     BLOCK_M = 1
     BLOCK_N = sympy.Min(N, 256)
-    ELEMS_PER_THREAD = BLOCK_N / wave_size
+    ELEMS_PER_THREAD = BLOCK_N

```

Comment:
Why would we want such large number for elements per thread though?

Reply:
Yes, you are right, we don't need this. Will update the PR.